### PR TITLE
scrollbar

### PR DESCRIPTION
**Pull request by MennaMagdy2019**
I made the scrollbar,scrollbar-track  with box-shadow and border-radius scrollbar-thumb and on hover on the scrollbar-thumb change the color of thumb

### CODE REVIEW

**Comment by MennaMagdy2019** [comment]
can you review it, please @fave77?

**Comment by apurv-1** [comment]
@MennaMagdy2019, can you show how the scroll bar looks now?
Additionally, I can see you've used WebKit, can you check if still works on browsers like Mozilla or Safari?

**Comment by MennaMagdy2019** [comment]
`It look likes now as
beforhover1
and on hover, It looks likes as
afterhover1

yes I checked it on safari and still works
@apurv-1

**Comment by MennaMagdy2019** [comment]
Are you prefer another color tho the scroll bar or another style? @apurv-1 @fave77

File: client/src/App.css [hunk 1]
```diff
@@ -19,3 +19,27 @@ code {
...
+/* Track */
+::-webkit-scrollbar-track {
+  box-shadow: inset 0 0 6px f1f1f1;
+  border-radius: 10px;
+  background: #f1f1f1;
+}
+
+/* Handle */
+::-webkit-scrollbar-thumb {
+  background: #7E7F81;
```
Comment (fave77) [inline client/src/App.css]: use this color: `#8299fd`

File: client/src/App.css [hunk 2]
```diff
@@ -19,3 +19,27 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+/***scrollbar***/
+/* width */
+::-webkit-scrollbar {
+  width:13px;
```
Comment (fave77) [inline client/src/App.css]: make the width 8px

**Review by fave77 — CHANGES_REQUESTED** [review]

**Comment by MennaMagdy2019** [comment]
the changes is Done 
the width is:8px and the color of scrollbar-thumb is #8299fd

modified
Is it Ok? @fave77

**Review by fave77 — CHANGES_REQUESTED** [review]
@MennaMagdy2019 the requirements have been met 👍🏻 ... just include changes only applicable for this issue i.e. `App.css`

Will merge it then!

File: client/src/components/contrib/contrib.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ import React, { useState, useEffect } from 'react';
 import ContribCard from './contrib-card.js';
 import FadeIn from 'react-fade-in';
 
+import {useWindowScroll} from "react-use";
```
Comment (apurv-1) [inline client/src/components/contrib/contrib.js]: Where are you using this `useWindowScroll`?

**Review by apurv-1 — CHANGES_REQUESTED** [review]
What you can do is `git add App.css` instead of `git add .`
And please mention the issue, like `fixes #119`

File: client/src/components/contrib/contrib.js (hunk 3 above)
Comment (MennaMagdy2019) [reply client/src/components/contrib/contrib.js]: I tried to make another thing and sorry I forgot to remove it but I didn't use it in this issue  @apurv-1

**Comment by MennaMagdy2019** [comment]
I did the final changes and I am sorry for adding other things @fave77 @apurv-1 
Are you need other changes?

**Review by MennaMagdy2019 — COMMENTED** [review]
I finished these changes

**Review by fave77 — CHANGES_REQUESTED** [review]
@MennaMagdy2019 please include the changes limited to `App.css` for this PR. 

The entire point for creating a new branch for each issue being that you can work on multiple issues simultaneously without making the respective changes convoluted.

**Comment by MennaMagdy2019** [comment]
Can I delete this branch and make another branch? @fave77

**Pull request closed without merging**